src/firefly/js/templates/fireflyviewer/FireflyViewer.js

```diff
@@ -118,19 +119,27 @@ FireflyViewer.defaultProps = {
     views: 'images | tables | xyplots'
 };
 
-function onReady({menu, options={}, normalInit}) {
+function onReady({menu, options={}, normalInit, appTitle}) {
     if (menu) {
         const {backgroundMonitor= true}= options;
         dispatchSetMenu({menuItems: menu, showBgMonitor:backgroundMonitor});
     }
     const {hasImages, hasTables, hasXyPlots} = getLayouInfo();
     if (normalInit && (!(hasImages || hasTables || hasXyPlots))) {
         let goto = getActionFromUrl();
-        if (!goto) {
-            const landingItem= menu.find( (item) => item.landing);
-            goto= landingItem && {type:SHOW_DROPDOWN, payload:{view:landingItem.action}};
+
+        const landingItem= menu.find( (item) => item.landing);
```

Comment:
`item.landing` should be documented here: https://github.com/Caltech-IPAC/firefly/blob/0d534c5533d8fe8aa9602b546d1ed86ac64b1afb/src/firefly/js/core/core-typedefs.jsdoc#L68

Reply:
I'm not sure how this works tbh, I just kept the old code (as you can tell from the diff)